-1

Tested at: b6acc69ce9671807e3af5ebdd0c74a008cdeb14a

CMSSW: CMSSW_11_2_X_2020-08-23-2300
SCRAM_ARCH: slc7_amd64_gcc820
You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-6a2320/8874/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found compilation error when building: 

<pre>>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-08-23-2300/src/CommonTools/RecoAlgos/plugins/VertexRefSelector.cc
>> Compiling edm plugin /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-08-23-2300/src/CommonTools/RecoAlgos/plugins/VertexSelector.cc
>> Building  edm plugin tmp/slc7_amd64_gcc820/src/CommonTools/RecoAlgos/plugins/CommonToolsRecoAlgos_plugins/libCommonToolsRecoAlgos_plugins.so
/cvmfs/cms-ib.cern.ch/nweek-02643/slc7_amd64_gcc820/external/gcc/8.2.0-bcolbf/bin/../lib/gcc/x86_64-unknown-linux-gnu/8.4.0/../../../../x86_64-unknown-linux-gnu/bin/ld: tmp/slc7_amd64_gcc820/src/CommonTools/RecoAlgos/plugins/CommonToolsRecoAlgos_plugins/ME0MuonTrackCollProducer.cc.o: in function `ME0MuonTrackCollProducer::produce(edm::Event&, edm::EventSetup const&)':
ME0MuonTrackCollProducer.cc:(.text+0x512): undefined reference to `muon::me0::isGoodMuon(reco::ME0Muon const&, muon::me0::SelectionType)'
collect2: error: ld returned 1 exit status
gmake: *** [tmp/slc7_amd64_gcc820/src/CommonTools/RecoAlgos/plugins/CommonToolsRecoAlgos_plugins/libCommonToolsRecoAlgos_plugins.so] Error 1
Leaving library rule at src/CommonTools/RecoAlgos/plugins
Entering library rule at CommonTools/RecoAlgos
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-08-23-2300/src/CommonTools/RecoAlgos/src/ClusterStorer.cc
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_2_X_2020-08-23-2300/src/CommonTools/RecoAlgos/src/MassiveCandidateConverter.cc
</pre>

--- a/CommonTools/RecoAlgos/BuildFile.xml
+++ b/CommonTools/RecoAlgos/BuildFile.xml
@@ -10,6 +10,7 @@
 <use name="DataFormats/RecoCandidate"/>
 <use name="DataFormats/Candidate"/>
 <use name="DataFormats/VertexReco"/>
+<use name="RecoMuon/MuonIdentification"/>
 <use name="TrackingTools/IPTools"/>
 <use name="TrackingTools/TransientTrack"/>
 <use name="fastjet"/>

